plugins/kotlin/fir/src/org/jetbrains/kotlin/idea/completion/contributors/FirCallableCompletionContributor.kt

```diff
@@ -153,7 +170,7 @@ internal open class FirCallableCompletionContributor(
             .filter { with(visibilityChecker) { isVisible(it) } }
             .filter { filter(it) }
             .forEach { callable ->
-                addCallableSymbolToCompletion(expectedType, callable, getOptions(callable))
+                addCallableSymbolToCompletion(expectedType, callable, getOptions(callable) ?: return@forEach, getPriority(callable))
```

Comment:
Please, do not use inline return/break/continue statements. Introduce a variable for that